### PR TITLE
[IMP] theme_clean: improve theme

### PR DESCRIPTION
**Pull request by xlu-odoo**
task-4178066
Part of task-4177975

### CODE REVIEW

File: theme_clean/views/snippets/s_company_team.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team" inherit_id="website.s_company_team">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt0 pb16" remove="pt48 pb48" separator=" "/>
```
Comment (SergeBayet) [inline theme_clean/views/snippets/s_company_team.xml:7]: Is the padding top is what we want?
image

File: theme_clean/views/snippets/s_numbers_showcase.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<template id="s_numbers_showcase" inherit_id="website.s_numbers_showcase">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc o_cc4 pt88" remove="pt80" separator=" "/>
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/04_001","colors":{"c3": "#FFFFFF"},"flip":["y"]}</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('container')]" position="before">
+        <div class="o_we_shape o_web_editor_Origins_04_001" style="background-image: url('/web_editor/shape/web_editor/Origins/04_001.svg?c3=o-color-4&amp;flip=y');"/>
+    </xpath>
+    <!-- Shape -->
```
Comment (SergeBayet) [inline theme_clean/views/snippets/s_numbers_showcase.xml:13]: This comment is misplaced or unnecessary.

File: theme_clean/views/snippets/s_title.xml [hunk 3]
```diff
@@ -2,30 +2,16 @@
 <odoo>
 
 <template id="s_title" inherit_id="website.s_title">
-    <!-- Section -->
+    <!-- Section-->
```
Comment (SergeBayet) [inline theme_clean/views/snippets/s_title.xml:5]:
```suggestion
    <!-- Section -->
```

**Review by SergeBayet — CHANGES_REQUESTED** [review]
@xlu-odoo Nice job :+1: , some remarks and nitpicking and we're good to go.

File: theme_clean/views/snippets/s_company_team.xml (hunk 1 above)
Comment (xlu-odoo) [reply theme_clean/views/snippets/s_company_team.xml:6]: ok snippet by default has now 0 custo (thus is pt48 pb48 padding), but template for configurator has now the changes I want for the page that will come out from the configurator.

**Comment by bso-odoo** [comment]
@robodoo delegate=SergeBayet

**Comment by SergeBayet** [comment]
@robodoo r+

**Pull request closed without merging**